src/Main/App.js

```diff
@@ -78,7 +78,11 @@ class App extends Component {
   }
 
   getPlayerFromReport(report, playerName) {
-    return report.friendlies.find(friendly => friendly.name === playerName);
+    const fetchByNameAttempt = report.friendlies.find(friendly => friendly.name === playerName);
+    if (!fetchByNameAttempt) {
+      return report.friendlies.find(friendly => friendly.id === parseInt(playerName, 10));
```

Comment:
`parseInt()` is outdated JS, please use `Number()` instead.

Reply:
I originally was going to include fight name and player name, but it requires blocking on the report be loaded which is certainly possible, but felt less clean. I can certainly try that route again.

I'll update to use Number().